### PR TITLE
Adding a file about fuquay varina

### LINKED ISSUE #42: Create a new file for githubteacher hometown
## Add file for Fuquay NC
- [x] Create a branch
- [x] Add my file
- [x] Create a commit
- [x] Open a pull request
- [x] Have a discussion about the pull request
- [x] Merge our pull request

### PR DESCRIPTION
**Pull request by githubteacher**
Which happens to be @githubteacher's hometown.

This Pull Request closes #42.

### CODE REVIEW

File: places/fuquay-varina.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Fuquay-Varina, NC
+
+### Great Places to Eat
```
Comment (githubteacher) [inline places/fuquay-varina.md:3]: @githubteacher I :heart: Italian - do you have any recommendations?

File: places/fuquay-varina.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+## Fuquay-Varina, NC
```
Comment (hectorsector) [inline places/fuquay-varina.md:1]: @githubteacher is this a real place??

Comment (tomiy1) [inline places/fuquay-varina.md:1]: @githubteacher San Diego has nicer weather! :+1:

Comment (githubteacher) [inline places/fuquay-varina.md:1]: > San Diego has nicer weather!

@tomiy1 completely agree!

Comment (githubteacher) [inline places/fuquay-varina.md:1]: > is this a real place??

@halfaro1 yes, I promise it is!

**Comment by atabh1** [comment]
git checkout master
git merge --no-ff githubteacher-hometown
git push origin master

**Comment by hectorsector** [comment]
@githubteacher 🚢

**Comment by tomiy1** [comment]
@githubteacher Looks good to go! :tada:

**Pull request merged**